zk.js/src/test-utils/test-rpc.ts

```diff
@@ -132,7 +149,7 @@ export class TestRpc extends Rpc {
     /** We mock the internal relayer server logic and must init a provider with the relayerKeypair */
     provider = await Provider.init({
       wallet: this.rpcKeypair,
-      rpc: this,
+      rpc: this as any,
```

Comment:
casting to a type (and especially to `any`) is very dangerous, let's not do that. 

To resolve the type error, you can align the getMerkleRoot method signature in the TestRpc class with the expected signature in the Rpc class. (TestRpc expects a PublicKey param, but Rpc expects none (just doing _merkleTreePubkey: PublicKey should suffice i think)